src/main/java/bhs/devilbotz/commands/arm/ArmDown.java

```diff
@@ -45,6 +45,6 @@ public void end(boolean interrupted) {
   // Returns true when the command should end.
   @Override
   public boolean isFinished() {
-    return arm.isBottomLimit() || arm.atBottom();
+    return arm.isBottomLimit();
```

Comment:
the `atBottom()` condition was causing the gripper arm to behave erratically when the bottom limit switch was broken/loose.